From 64ae3741c86f5740437480d60cec9c1f8ddb5158 Mon Sep 17 00:00:00 2001
From: Florinel Iordache <florinel.iordache@nxp.com>
Date: Mon, 5 Nov 2018 17:03:04 +0200
Subject: [PATCH 757/767] arm64: dts: lx2160: DPMAC connections to backplane
 PHYs example

This is an example of device tree nodes required to enable 10GBase-KR and 40GBase-KR on LX2160

Signed-off-by: Florinel Iordache <florinel.iordache@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 .../boot/dts/freescale/fsl-lx2160a-qds.dts    | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts
index 311a84a4d746..93abcb908cf9 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts
@@ -150,3 +150,23 @@
 		fsl,lane-reg = <0xE00 0x100>; /* lane G */
 	};
 };
+
+/* Update DPMAC connections to 40G backplane PHYs
+ * &dpmac1 {
+ *	phy-handle = <&pcs_phy1>;
+ * };
+ * 
+ * &dpmac2 {
+ *	phy-handle = <&pcs_phy2>;
+ * };
+ */
+
+/* Update DPMAC connections to 10G backplane PHYs
+ * &dpmac3 {
+ *	phy-handle = <&pcs_phy3>;
+ * };
+ * 
+ * &dpmac4 {
+ *	phy-handle = <&pcs_phy4>;
+ * };
+ */
-- 
2.17.0

